.circleci/config.yml

```diff
@@ -162,6 +162,16 @@ jobs:
       - store_artifacts:
           path: ~/action/build
           destination: build
+      - run:
+          name: Upload sentry artifacts
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "production" ]; then
+              curl -sL https://sentry.io/get-cli/ | bash
```

Comment:
Simple enough...they didn't have a Docker container mixin?

Reply:
they do! want me to switch?

Reply:
Could you? It should speed deploys a little bit: CircleCI should cache the container